src/tensors/gpu/tensor_operators.cu

```diff
@@ -1246,7 +1175,9 @@ __global__ void gCrossEntropyPick(float* out,
   }
 }
 
-void CrossEntropyPick(Tensor out, Tensor in, Tensor pick) {
+void CrossEntropyPick(Tensor out, Tensor in, Tensor indices) {
```

Comment:
From the name I cannot tell what this does. Add a very brief comment?

Reply:
Copied over comment from Node.